pkg/controller/clusterautoscaler/clusterautoscaler_test.go

```diff
@@ -145,7 +146,8 @@ func TestAutoscalerArgs(t *testing.T) {
 
 	expectedMissing := []string{
 		"--scale-down-delay-after-delete",
-		"--scale-down-delay-after-failure",
```

Comment:
i'm not understanding why this line is being removed

Reply:
No this looks like a mistake, thanks - will fix!